host-controller/src/main/java/org/jboss/as/host/controller/ManagedServerBootCmdFactory.java

```diff
@@ -239,6 +242,8 @@ public HostControllerEnvironment getHostControllerEnvironment() {
 
         command.add("-D[" + ManagedServer.getServerProcessName(serverName) + "]");
 
+        propagateJavaModuleParams(command);
```

Comment:
Sorry for delay to reply to your comment.
I'll exclude these 'unsafe' changes from my PR @bstansberry .
Until JDK9 is not in final state it is hard to say what final solution will be.
According to latest discussions on jigsaw-dev mailing list it seems 
this change will not be necessary at the end. But things are still evolving there ...